src/components/QueryCondition.vue

```diff
@@ -122,6 +123,24 @@ export default Vue.extend( {
 				this.$store.dispatch( 'updateValue', { value, conditionIndex: this.conditionIndex } );
 			},
 		},
+		negate: {
```

Comment:
You can do that with one computed property. Having those two seems a bit too much

Reply:
Yeah, it grew organically when I was writing the code. Fixed now.

Reply:
:+1: